charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml

```diff
@@ -150,8 +150,10 @@ spec:
       affinity:
 {{ toYaml .Values.vminsert.affinity | indent 8 }}
     {{- end }}
-{{- end }}
+    {{- if .Values.vminsert.extraVolumes }}
       volumes:
         {{- with .Values.vminsert.extraVolumes }}
           {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- end }}
+{{- end }}
```

Comment:
you add one `if` but two `end` is it correct?

Reply:
Yes, there was an `end` missing in the previous config so added that at line 159 and also the `end` at 158 is for the `if `condition.